source/settings/core.rst

```diff
@@ -2974,6 +2974,45 @@ See :ref:`settings` for list of all setting groups.
 
    The static password to be used for all users authenticating using this passdb.
 
+.. dovecot_core:setting:: passdb_pam_max_requests
+   :default: 100
+   :values: @uint
+   :added: 3.0.0
+
+   Number of requests a auth-worker process handles for passdb pam before
+   it dies.
+
+.. dovecot_core:setting:: passdb_pam_failure_show_msg
+   :default: no
+   :values: @boolean
+   :added: 3.0.0
+
+   Replace the default ``Authentication failed`` reply with PAM's failure.
+
+.. dovecot_core:setting:: passdb_pam_service_name
+   :default: dovecot
+   :values: @string
+   :added: 3.0.0
+
+   The pam service name to be used with the pam passdb.
+
+
+.. dovecot_core:setting:: passdb_pam_setcred
+   :default: no
+   :values: @boolean
+   :added: 3.0.0
+
+   Make Dovecot create PAM credentials.
```

Comment:
Add also: The credentials are never deleted, which may cause problems with some PAM plugins.